### ISSUE TITLE
Привести формы клиента в соответствие с макетами. 

### ISSUE THREAD

**Issue by hodkovdd** [sedovalx/taxi#68]
@Argelein 
1. Сделать выше строки в таблицах чтобы ячейки были больше похожи на квадраты.
2. Отрицательные значения отображались красным цветом.

**Comment by sedovalx** [comment]
@hodkoff2 первое зачем? Это вряд ли возможно, т.к. используемый компонент таблицы по-моему не поддерживает настраиваемую высоту строки. Это позволяет этой таблице работать в виртуальном режиме, отрисовывая только те строки, которые видны пользователю, что крайне положительно сказывается на производительности.

**Comment by hodkovdd** [comment]
@sedovalx 
ок, а второе?

**Comment by sedovalx** [comment]
@hodkoff2 да, это можно, думаю. @Argelein делается примерно так же, как сделана галочка в кассовой форме.

**Comment by hodkovdd** [comment]
Когда сделаешь?  @Argelein

**Comment by kirzas** [comment]
@sedovalx 
чего-то я не соображу, как это сделать
Где задается класс check-icon-cell для галочки? Поиск не ищет.  если я увижу, как он определен, может быть смогу сделать.

Я знаю только как по простому сделать: сделать класс, который имеет красный цвет и применять его только если значение меньше 0. Я не могу понять, где у нас сейчас классы определены, у нас же готовый набор стилей был взят. К тому же я не понимаю, как такую конструкцию вставить в ember.table...
нужна помощь с этим.

**Comment by sedovalx** [comment]
@Argelein смотри \sources\client\app\views\check-icon-cell.js
